crates/sui-framework/packages/sui-framework/sources/coin.move

```diff
@@ -217,23 +367,7 @@ public fun create_currency<T: drop>(
     icon_url: Option<Url>,
     ctx: &mut TxContext,
 ): (TreasuryCap<T>, CoinMetadata<T>) {
-    // Make sure there's only one instance of the type T
-    assert!(sui::types::is_one_time_witness(&witness), EBadWitness);
-
-    (
-        TreasuryCap {
-            id: object::new(ctx),
-            total_supply: balance::create_supply(witness),
-        },
-        CoinMetadata {
-            id: object::new(ctx),
-            decimals,
-            name: string::utf8(name),
-            symbol: ascii::string(symbol),
-            description: string::utf8(description),
-            icon_url,
-        },
-    )
+    abort 1337
```

Comment:
I think just a bare `abort` might work these days? In case it doesn't though, we should give this a proper error code `1337` is really only for tests.